metrix/src/app.rs

```diff
@@ -108,13 +109,19 @@ fn aggregate_metrics_route(
         filter_clause
     );
     let results = sql_query(query_string)
-        .load(&db_conn)
+        .load::<Bucket>(&db_conn)
         .expect("Error loading metrics");
 
+    let mut padded_results: Vec<Bucket> = vec![();bucket_count as usize]
```

Comment:
isn't bucket_size a float? Does the type cast turn it into an int?

Reply:
Bucket count. Not buck size